themes/openy_themes/openy_rose/templates/paragraph/paragraph--grid-content--default.html.twig

```diff
@@ -0,0 +1,64 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a paragraph.
+ *
+ * Available variables:
+ * - paragraph: Full paragraph entity.
+ *   - id: The paragraph ID.
+ *   - bundle: The type of the paragraph, for example, "image" or "text".
+ *   - authorid: The user ID of the paragraph author.
+ *   - createdtime: Formatted creation date. Preprocess functions can
+ *     reformat it by calling format_date() with the desired parameters on
+ *     $variables['paragraph']->getCreatedTime().
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{%
+set classes = [
+  'paragraph',
+  'row-eq-height',
+  'paragraph--column-in-a-grid',
+  'paragraph--type--' ~ paragraph.bundle|clean_class,
+  view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
+]
+%}
+{% if content.field_prgf_grid_style.0['#markup'] == '2' %}
+  {% set item_class = 'col-xs-12 col-sm-6 row-eq-height' %}
+{% elseif content.field_prgf_grid_style.0['#markup'] == '3' %}
+  {% set item_class = 'col-xs-12 col-sm-4 row-eq-height' %}
+{% elseif content.field_prgf_grid_style.0['#markup'] == '4' %}
+  {% set item_class = 'col-xs-12 col-sm-2 row-eq-height' %}
```

Comment:
It should be `col-sm-3` instead of `col-sm-2`, because 12/4 = 3. Otherwise it looks like this:
![image](https://cloud.githubusercontent.com/assets/1316234/22456796/4fd09208-e79d-11e6-89f0-1e3847fc539a.png)